configfiles/application-default.yml

```diff
@@ -17,6 +17,8 @@ files:
     location:
       static: C:/fincity/resources/static
       secured: C:/fincity/resources/secured
+  secureKeyURI: /api/files/secured/downloadFileByKey/      
```

Comment:
Remove this and default it like I showed you yesterday, it is not going to change easily

Reply:
Also it should not start with /

Reply:
done.